helm/hpcc/values.schema.json

```diff
@@ -1196,6 +1205,34 @@
           "type": "object"
         }
       }
+    },
+    "sinks": {
+      "type": "array",
+      "items" : {
+        "$ref" : "#/definitions/sink"
+      }
+    },
+    "sink": {
+      "type" : "object",
+      "description" : "Definition of a metric sink",
+      "properties" : {
+        "type": {
+          "type" : "string",
+          "description" : "The defined metric sink type"
+        },
+        "name" : {
```

Comment:
I'm curious what the name is for.  It is required, but doesn't seem to serve any purpose as far as the user is concerned.

Reply:
I'm not convinced it should be required (or even be present).  The helm charts are using lists for the metrics so there isn't a way to override the global metrics, except for completely replacing them.
I think if the framework needs to differentiate the sinks then it should be done internally.
Not significant enough not to merge this PR though, but should be revisited.